source/_integrations/thread.markdown

```diff
@@ -14,30 +14,144 @@ ha_platforms:
+ </p>
 
-Each vendor forms their own network when you start using their products, so you can end up having a Home Assistant, an Apple, and a Google Thread network in your home. These are all separate networks using different credentials, which prevents devices from roaming between TBRs.
+Matter and Apple HomeKit are smart home protocols. They are responsible for handling the Thread credentials and connecting your Thread device to the Thread network. A smart home protocol is needed to control your device. Both home automation standards are supported natively by Home Assistant.
 
-It is possible to align credentials for TBRs from different vendors and have them form a single network together. This allows you to freely move devices between rooms without losing connectivity. To do this, you need to make sure that all TBRs use the same credentials. Currently, this works differently for each TBR vendor.
+## Adding a Thread-based device to Home Assistant
 
-Home Assistant will sync the Thread credentials with Google when starting to {% term commission %} a Matter device via the Home Assistant Companion app. For other vendors, if the vendor allows you to see the operational dataset in TLV format, you can import it to Home Assistant from the Thread panel.
+How a Thread-based device is added to Home Assistant depends on the home automation standard it uses.
+
+1. If you see the Matter logo on your device, follow the procedure [adding a Matter device to Home Assistant](/integrations/matter/#adding-a-matter-device-to-home-assistant).
+
+   <img src="/images/integrations/thread/matter_onpackbadge_logo.png"  width="200">
+
+2. If you see the Apple HomeKit logo on your device, follow the procedure [adding a HomeKit device to Home Assistant](/integrations/homekit_controller/#adding-a-homekit-device-through-thread).
+
+   <img src="/images/integrations/thread/apple-works-with-homekit-logo.png"  width="200">
+
+## About Thread
+
+This section introduces the terms *Thread* and *border router* and lists border routers that are supported by Home Assistant.
+
+### A communication protocol for IoT devices
+
+Thread is a low-power mesh networking standard for IoT devices. The low-power aspect is important for battery-powered smart home devices. However, it's also low-bandwidth, making it ideal for applications that don't send a lot of data, like switches or motion sensors.
+
+Thread uses the same <abbr title="radio frequency">RF</abbr> technology as Zigbee (IEEE 802.15.4) but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread does not allow controlling devices: It is just a communication protocol. To control the Thread devices, a higher-level protocol is required: Matter or Apple HomeKit.
+
+Thread devices use the IPv6 standard to communicate both inside and outside the mesh network.
```

Comment:
I don't think this needs a separate paragraph. Also small wording improvement ("Thread by itself does not allow")

```suggestion
Thread uses the same <abbr title="radio frequency">RF</abbr> technology as Zigbee (IEEE 802.15.4) but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread by itself does not allow controlling devices: It is just a communication protocol. To control the Thread devices, a higher-level protocol is required: Matter or Apple HomeKit. Thread devices use the IPv6 standard to communicate both inside and outside the mesh network.
```